code/modules/mob/living/simple_animal/abnormality/teth/punishing_bird.dm

```diff
@@ -91,7 +91,7 @@
 	if(..())
 		if(obj_damage > 0) // Already transformed
 			return
-		if(prob(4))
+		if(prob(20))
```

Comment:
PBird code is being changed in #379

Reply:
Will definitely change once the PR is merged. I'll be fixing a lot of merge conflicts then but I'm fine with that.